proselint/command_line.py

```diff
@@ -27,6 +27,7 @@
 demo_file = os.path.join(proselint_path, "demo.md")
 
 
+# TODO: fix broken corpus
```

Comment:
FYI, the corpus can be found at https://github.com/amperser/corpus, but because we don't have the rights to distribute the texts, it's a private repo. I'll give you access soon. If you can think of a workaround that allows us to keep the test, let me know.

Reply:
Interesting. I cannot think of a workaround at the moment, however, it is an intriguing feature. I must ask, though, what is the point in including the option in the end software if it is broken by default without the corpus and people cannot obtain it?

Reply:
There’s no reason to include it in the end software, could splinter it off into a developer only tool or have it appear only in tests

Reply:
That'd make more sense. In that case, I'll remove the option altogether for now.

Reply:
Great, thanks. Can you open an issue for it?